public/static/locales/en/pageTitles.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Shouldn't these be English: English, so that the keys for each of the two pageTitles.jsons have the same keys?

Reply:
No, this is to fix a weird issue with the page titles rendering before the javascript on the page loads.  The Page titles need to be translated server side to avoid them rendering english first then changing to welsh.  This means the english keys need to be welsh to allow client side to switch back and forth.
I would have added a comment to the top of the file but then the i18n package complained that it wasn't valid JSON.  There is a comment explaining in the PageTitles.js helper instead.